### ISSUE TITLE
Port to other platforms

### ISSUE THREAD

**Issue by saagarjha** [saagarjha/MacCast#18]
Stream to iOS, cast from Windows/Linux, …

**Comment by Geczy** [comment]
Yes please, Windows support would be great, what are the steps to make that work exactly? A rough outline

**Comment by saagarjha** [comment]
* Write a new communication protocol that works on Windows and visionOS
* Write a Windows client that handles all the things the Mac app does (recording windows, forwarding events)

This is probably a couple weeks of effort, and I don't use Windows, so I don't plan to look into this right now.

**Comment by billyjacoby** [comment]
If this is the route this project ends up going I think something like Tauri would be a good candidate to build the host application in. This would make the maintenance of the host app much simpler since the code could be shared between Windows and Mac, and I've actually run a Swift library in Tauri before as well.

**Comment by saagarjha** [comment]
There's not a whole lot of UI that needs to be written (you can see how little is in the Mac app!) It's more that you'd need OS-specific code to handle plumbing that cannot be easily be done in a cross-platform framework.
